src/app/item-popup/ItemPopupHeader.tsx

```diff
@@ -151,11 +151,11 @@ function destinyDBLink(item: DimItem) {
     const perkCsv = buildPerksCsv(d2Item);
 
     if (perkCsv && perkCsv.length > 0) {
-      perkQueryString = `&perks=${perkCsv}`;
+      perkQueryString = `?perks=${perkCsv}`;
     }
   }
 
-  return `https://destinytracker.com/destiny-2/db/items/${item.hash}${perkQueryString}`;
+  return `https://destinytracker.com/destiny-2/db/items/${item.hash}${perkQueryString ? perkQueryString : ''}`;
```

Comment:
I should have just init'd `perkQueryString` to a sane (empty string) default instead of making us jump through hoops here.